### DISCUSSION TITLE
I18n: Add browser language detector

### DISCUSSION THREAD

**Discussion by pbaumard** [grafana/grafana#71009] [General]
This is a discussion for PR #69396 which adds a browser language detector for the default language if default_language server setting is changed.

A browser language detector is much better for user experience instead of setting the language in user settings.

The proposed doc is:

> ### default_language
> 
> If a supported IETF language tag like `en-US` is available, this option will set the default UI language.
> If this setting is not a supported language tag, such as `detector`, the default UI language will be determined by a browser detector (see [default detector options](https://github.com/i18next/i18next-browser-languageDetector#detector-options)).
> The default is `en-US`.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is phrased as a discussion, and we'd like open discussions about grafana to go to our[ community forum](https://community.grafana.com/).
